### ISSUE TITLE
More explanation about the solution

### ISSUE THREAD

**Issue by llahti** [llahti/CarND-Traffic-Sign-Classifier-Project#6]
How would I choose the optimizer? What is its Pros & Cons and how would I evaluate it?
How would I decide the number and type of layers?
How would I tune the hyperparameter? How many values should I test and how to decide the values?
How would I preprocess my data? Why do I need to apply a certain technique?
How would I train the model?
How would I evaluate the model? What is the metric? How do I set the benchmark?
